pkg/controller/openpitrix/helmrelease/helm_release_controller.go

```diff
@@ -339,7 +338,6 @@ func (r *ReconcileHelmRelease) createOrUpgradeHelmRelease(rls *v1alpha1.HelmRele
 
 	// If clusterConfig is empty, this application will be installed in current host.
 	hw := helmwrapper.NewHelmWrapper(clusterConfig, rls.GetRlsNamespace(), rls.Spec.Name,
-		helmwrapper.SetAnnotations(map[string]string{constants.CreatorAnnotationKey: rls.GetCreator()}),
```

Comment:
We can't simply remove this field, front end needs this value to display the creator of the app, you can change the key to apps.kubesphere.io/creator.